client.py

```diff
@@ -0,0 +1,30 @@
+import sys
+import socket
+
+
+def client(message):
+    if not isinstance(message, str):
+        message = str(message)
```

Comment:
Watch out for this one.  What if the message is a unicode object containing characters not in the ASCII byte space?